src/main/kotlin/no/nav/familie/ef/sak/behandlingsflyt/steg/BeregnYtelseSteg.kt

```diff
@@ -164,6 +182,14 @@ class BeregnYtelseSteg(private val tilkjentYtelseService: TilkjentYtelseService,
 
     }
 
+    private fun andelerForSanksjonertRevurdering(behandling: Behandling,
+                                                 opphørsperiode: Periode): List<AndelTilkjentYtelse> {
+        return behandling.forrigeBehandlingId?.let {
+            val forrigeTilkjenteYtelse = hentForrigeTilkjenteYtelse(behandling)
+            return vurderPeriodeForOpphør(forrigeTilkjenteYtelse.andelerTilkjentYtelse, listOf(opphørsperiode))
+        } ?: throw Feil("Forsøk på å opprette sanksjon mislyktes")
```

Comment:
Kanskje en annen feilmelding her? 
```suggestion
        } ?: throw Feil("Kan ikke opprette sanksjon når det ikke finnes en tidligere behandling")
```

Hvis det er sånn att det ikke går å opprette sanksjon hvis det ikke finnes tidligere behandling da?

Reply:
Fixed i [87b4db9](https://github.com/navikt/familie-ef-sak/pull/1172/commits/87b4db979f71a0b7180acc19e44fb3898e683a69)